contents/blog/how-we-turned-clickhouse-into-our-eventmansion.md

```diff
@@ -0,0 +1,84 @@
+---
+date: 2021-11-08
+title: How we turned ClickHouse into our event mansion
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+Author: joe-martin
+featuredImage: ../images/blog/clickhouse-announcement.png
+featuredImageType: full
+---
+
+Recently, PostHog was invited to speak at OSA Con 2021, an open source analytics conference organised by Altinity. It was a fantastic opportunity to talk to the community, alongside other engineers and leaders from companies such as eBay, Facebook and Rudderstack. 
```

Comment:
I would probably drop these names over eBay and Facebook since they are more relevant to the space. People who know will definitely pick up these and go, 'oh, nice.'

```suggestion
Recently, PostHog was invited to speak at OSA Con 2021, an open source analytics conference organised by Altinity. It was a fantastic opportunity to talk to the community, alongside other engineers and leaders from projects such as ClickHouse, Airflow, Superset & Preset, Pinot, Druid, Cube.js, and Presto. 
```

Reply:
agreed especially given the audience